changedate + add VEP_UCD (#50)
ivoa/HighEnergyObsCoreExt: VEPforUCD/VEP-UCD-31-obs-event.txt

Vocabulary: http://www.ivoa.net/rdf/UCDlist/
Author: mireille.louys@unistra.fr, khelifi@in2p3.fr
Date: 2026-04-23
Nb of terms discussed together : 1
-----------------------

New Term: obs.event
Prefix: S
Label: Related to a signal messenger detection (photon or particle)

Description:  

Rationale: 
High energy experiences gather detections of messengers (photon or particle) as lists of events.

> An event can be recorded by various measurements like time of arrival, detector position, energy , etc. 

Reviewer: ```suggestion
An event can be associated with various measurements like time of arrival, detector position, energy, etc. 
```

Author: Ok for me .

An event can be categorized using properties like :  
event identifier : as meta.id;obs.event
event grade : as meta.code.class;obs.event. 
( defined from a detector pixel island, for instance)
event type: as meta.code.class;obs.event 

> (defined according to some decision in the analysis pipeline)

Reviewer: ```suggestion
- event identifier : as meta.id;obs.event
- event grade : as meta.code.class;obs.event. 
(defined from a detector pixel island, for instance)
- event type: as meta.code.class;obs.event 
(defined according to some decision in the analysis pipeline)
```

Reviewer: The strong problem with the workding "class" of meta.code.class is that in HE and VHE, we have both classes and types, that are fundamentaly different. This mixes the concepts and will create many many confusion for the HE people.
The differences are:
- types divide the space space of events into non-overlapping subsets, such that the subsets can be statistically analysed because they form independant datasets (in term of statistics)
- classes divides the space space of events based on instrumental properties. The notion of class is (almost) identical to event grade
So, I strongly, very strongly recommand to use a new ucd called "meta.code.type", to really separate the concept!
PS: the event grade is using meta.code.class;obs.event, illustrating that the event grade is a different concept that event type
